src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuilder.java

```diff
@@ -133,7 +133,7 @@ private int getScmChange(AbstractProject subjob, PhaseJobsConfig phaseConfig, Ab
 
     public boolean evalCondition(final String condition, final AbstractBuild<?, ?> build, final BuildListener listener) {
         try {
-            return (Boolean) Eval.me(expandToken(condition, build, listener).toLowerCase().trim());
+            return (Boolean) Eval.me(expandToken(condition, build, listener).trim());

```

Comment:
May break cases, which rely on the current behavior. Probably Eval should be made universal somehow